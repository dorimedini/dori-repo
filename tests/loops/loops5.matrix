{
	int f = 10;
	int g = 20;
	matrix m(1,1) = [4];
	int i;
	while (f > 0 || true && g > f)
	{
		print "iteration:";
		print i;
		{
			f = f - 3;
		}	
		g = g - 5;
		i = i + 1 * -1 * -1;
	}
	print f;
}

