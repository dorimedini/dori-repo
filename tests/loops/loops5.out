iteration:
0
iteration:
1
iteration:
2
iteration:
3
iteration:
4
-5
